runtime/planificator.js

```diff
@@ -5,6 +5,9 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+// Deprecated.
+// TODO: Replace by ts/planificator.ts
```

Comment:
If you delete this then git will properly mark this as a file move.

Reply:
this file was refactored into several pieces, so not a lot of value in move, but like I said in the other comment, I'd like to keep the original planificator around for just a bit.

Reply:
Looks like it's just github not using -C for diffs to detect copies...  ah well.